openApiDocs/v1.0/ChangeNotifications.yml

```diff
@@ -13,50 +13,7 @@ paths:
       summary: Get entities from subscriptions
       operationId: subscriptions.subscription_ListSubscription
       parameters:
-        - $ref: '#/components/parameters/top'
```

Comment:
This a breaking change. Related to https://github.com/microsoftgraph/microsoft-graph-devx-api/issues/611

Reply:
This is also broken due property typo in `getApplicablePolicyRequirements` action. cc\ @darrelmiller.